packages/magellan-mocha-plugin/lib/get_tests.js

```diff
@@ -26,16 +30,14 @@ module.exports = function ( settings ) {
 
 	args.push( '--reporter', reporter );
 
-	/* istanbul ignore else */
-	if ( mochaSettings.mochaOpts ) {
-		args.push( '--opts', mochaSettings.mochaOpts );
+	if ( mochaSettings.mochaConfig ) {
```

Comment:
Mocha 8.0 dropped support for `--opts`, instead they recommend to use `--config`.